extensions/postgres/schema/xdb_sys_schema.sql

```diff
@@ -37,15 +37,23 @@ CREATE TABLE xdb_sys_async_state_executions(
    PRIMARY KEY (process_execution_id, state_id, state_id_sequence)
 );
 
-CREATE TABLE xdb_sys_worker_tasks(
+CREATE TABLE xdb_sys_immediate_tasks(
     shard_id INTEGER NOT NULL, -- for sharding xdb-local-mq
     task_sequence bigserial,   
     --
-    task_type SMALLINT NOT NULL, -- 1: waitUntil 2: execute
-    process_execution_id uuid, -- for looking up xdb_sys_async_state_executions
```

Comment:
Here is the key schema modification to support local_queue_message by reusing the woker_tasks table.

Reply:
I feel like it's better to just rename the table as well. xdb_sys_worker_tasks ==> xdb_sys_immediate_tasks.

Reply:
Already renamed.

Reply:
ahh right. I misread it